Sources/eppo/EppoClient.swift

```diff
@@ -1,32 +1,30 @@
 import Foundation;
 
-public let version = "1.0.1"
-
-public struct FlagConfigJSON : Decodable {
```

Comment:
moved `FlagConfigJSON` struct to the `dto` directory with the rest of them - this object represents the "top level" of parsed flag configuration